application_store_api/namespace_fund/routes.py

```diff
@@ -0,0 +1,96 @@
+from application_store_api.namespace_fund.data_store.store import APPLICATIONS
+from application_store_api.namespace_fund.fund_ns import (
+    application_model_inbound,
+)
+from application_store_api.namespace_fund.fund_ns import fund_ns
+from flask_restx import reqparse
+from flask_restx import Resource
+
+
+# GET/DELETE all funds
+@fund_ns.route("/all_funds")
+class Fund(Resource):
+    query_params_parser = reqparse.RequestParser()
+    query_params_parser.add_argument(
+        "delete_key", type=str, help="key to clear data store"
+    )
+
+    @fund_ns.doc("list_funds")
+    def get(self):
+        return APPLICATIONS.get_funds()
+
+    # Remove once data store is persisted outside of application memory
+    @fund_ns.doc("list_funds", parser=query_params_parser)
+    def delete(self):
+        args = self.query_params_parser.parse_args()
+        delete_key = args["delete_key"]
+        return APPLICATIONS.delete_all(delete_key)
+
+
+# POST a new application
+@fund_ns.route("/new_application")
+class NewApplication(Resource):
+    @fund_ns.doc("create_application")
+    @fund_ns.expect(application_model_inbound)
+    @fund_ns.marshal_with(application_model_inbound, code=201)
+    def post(self):
```

Comment:
Again, this is a comment about flask_restx rather than this code, but the requirement to use lots of decorators (with perhaps not totally clear method names) seems to make it harder to read, and perhaps write? Worth discussing ways to simplify?